packages/contracts-bedrock/package.json

```
@@ -17,19 +17,19 @@
     "build:forge": "forge build",
     "build:differential": "go build -o ./scripts/differential-testing/differential-testing ./scripts/differential-testing",
     "build:fuzz": "(cd test-case-generator && go build ./cmd/fuzz.go)",
-    "autogen:invariant-docs": "ts-node scripts/invariant-doc-gen.ts",
+    "autogen:invariant-docs": "tsx scripts/invariant-doc-gen.ts",
     "test": "pnpm build:differential && pnpm build:fuzz && forge test",
     "coverage": "pnpm build:differential && pnpm build:fuzz && forge coverage",
     "coverage:lcov": "pnpm build:differential && pnpm build:fuzz && forge coverage --report lcov",
     "gas-snapshot": "pnpm build:differential && pnpm build:fuzz && forge snapshot --no-match-test 'testDiff|testFuzz|invariant|generateArtifact'",
     "storage-snapshot": "./scripts/storage-snapshot.sh",
     "validate-deploy-configs": "./scripts/validate-deploy-configs.sh",
-    "validate-spacers": "forge build && npx ts-node scripts/validate-spacers.ts",
+    "validate-spacers": "forge build && npx tsx scripts/validate-spacers.ts",
```

```suggestion
    "validate-spacers": "forge build && tsx scripts/validate-spacers.ts",
```
npx slows down tsx a lot from needing to do a lookup